lokoctl installation document (#77)
kinvolk/lokomotive: docs/installer/lokoctl.md

# lokoctl

lokoctl is a command line interface for Lokomotive, Kinvolk's open-source Kubernetes distribution
which includes installers for various platforms and Lokomotive components.

## Installation

### Download official releases

To install, find the [appropriate
package](https://github.com/kinvolk/lokomotive/releases) for your system and

> download it. Lokoctl is packaged as a `tar.gz` archive.

Reviewer: ```suggestion
download it. lokoctl is packaged as a `tar.gz` archive.
```

```console
wget <URL-TO-DOWNLOAD-LOKOCTL>
```
After downloading, untar the package. Lokoctl runs as a single binary named `lokoctl`

```console
tar xvzf <LOKOCTL_TAR_GZ_ARCHIVE>
```

The final step is to make sure that the `lokoctl` binary is available on the `PATH`

```console
export PATH=$PATH:/path/to/lokoctl/binary
```

### Using 'go get'

You can quickly get latest version of `lokoctl` by running following command:
```

> go get github.com/kinvolk/lokomotive/cmd/lokoctl

Reviewer 2: I know this is very trivial but saying that you should have golang installed on the system is important I think.

Author: I think this is implied.

```

Once finished, `lokoctl` binary should be available in `$GOPATH/bin`.

### Building from source

Clone this repository and build the lokoctl binary:

```bash
git clone https://github.com/kinvolk/lokomotive
cd lokomotive

> make

Reviewer 3: It should be noted, where the binary will be placed after running make.

# lokoctl

lokoctl is a command line interface for Lokomotive, Kinvolk's open-source Kubernetes distribution
which includes installers for various platforms and Lokomotive components.

## Installation

### Download official releases

To install, find the [appropriate
package](https://github.com/kinvolk/lokomotive/releases) for your system and
download it. lokoctl is packaged as a `tar.gz` archive.

```console
wget <URL-TO-DOWNLOAD-LOKOCTL>
```
After downloading, untar the package. Lokoctl runs as a single binary named `lokoctl`.

```console
tar xvzf <LOKOCTL_TAR_GZ_ARCHIVE>
```

The final step is to make sure that the `lokoctl` binary is available on the `PATH`.

```console
export PATH=$PATH:/path/to/lokoctl/binary
```

### Using 'go get'

You can quickly get latest version of `lokoctl` by running following command:
```
go get github.com/kinvolk/lokomotive/cmd/lokoctl
```

Once finished, `lokoctl` binary should be available in `$GOPATH/bin`.

### Building from source

Clone this repository and build the lokoctl binary:

```bash
git clone https://github.com/kinvolk/lokomotive
cd lokomotive
make
```

> The binary will be created in the same directory in which the `make` command was executed.

Reviewer 3: What if I do `make -C lokomotive` ? :wink: 

Maybe:
```suggestion
The binary will be created in the projects main directory.
```
